### PR TITLE
wish controller and some ui

### PR DESCRIPTION
**Pull request by Moshizzl3**
What kind of change does this PR introduce?

- [ ] `bug-fix`
- [x] `new-feature`
- [x] `enhancement`

### CODE REVIEW

File: src/main/resources/templates/createwish.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="https://www.thymeleaf.org">
+<head>
+    <di th:replace="fragments/head"></di>
+    <meta charset="UTF-8">
```
Comment (MathiasReker) [inline src/main/resources/templates/createwish.html:5]:
```suggestion
```

File: src/main/resources/templates/createwish.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    <title>Wish</title>
+</head>
+<body>
+<div th:replace="fragments/navbar"></div>
+<h1>Wish</h1>
```
Comment (MathiasReker) [inline src/main/resources/templates/createwish.html:10]: Move to inside section

File: src/main/resources/templates/index.html [hunk 3]
```diff
@@ -9,7 +9,7 @@
 <a href="/login">Login</a><br>
 <a href="/signup">Signup</a><br>
 <a href="/dashboard">Dashboard</a><br>
-
+<a href="createwish">Create wish</a>
```
Comment (MathiasReker) [inline src/main/resources/templates/index.html:12]: use kebab case for hyperlinks

File: src/main/java/com/group6/wishshare/web/WishController.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
+package com.group6.wishshare.web;
+
+import com.group6.wishshare.data.repository.WishRepository;
+import com.group6.wishshare.domain.model.Wish;
```
Comment (MathiasReker) [inline src/main/java/com/group6/wishshare/web/WishController.java:4]: use a service between the controller and the model :-)

File: src/main/resources/static/css/soft-ui-dashboard.css [hunk 5]
```diff
@@ -2774,7 +2774,7 @@ textarea.form-control-lg {
   display: flex;
   flex-wrap: wrap;
   align-items: stretch;
-  width: 100%;
+  width: 33%;
```
Comment (MathiasReker) [inline src/main/resources/static/css/soft-ui-dashboard.css:2777]: Don't edit the compiled CSS :) Why is this needed?

File: src/main/java/com/group6/wishshare/web/WishController.java [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+  WishService wishService = new WishService(new WishRepository());
+
+  @GetMapping("/create-wish")
+  public String createwish(Model model) {
+    List<Wish> wishes = wishService.getWishes(1);
+    model.addAttribute("wishes", wishes);
+    return "createwish";
+  }
+
+  @PostMapping("/new-wish")
```
Comment (MathiasReker) [inline src/main/java/com/group6/wishshare/web/WishController.java:26]: Looks good! :)

**Review by MathiasReker — CHANGES_REQUESTED** [review]
Cool! Just a few comments.

File: src/main/resources/templates/createwish.html (hunk 2 above)
Comment (MathiasReker) [inline src/main/resources/templates/createwish.html:10]: Wrap the h1 tag and the section inside a article tag: https://css-tricks.com/how-to-section-your-html/

**Review by Jarkyman — APPROVED** [review]

**Pull request merged**